components/md.js

```diff
@@ -85,6 +85,8 @@ const md = (strings, ...values) => {
           return <LiveEdit code={literal} noInline />
         } else if (language === 'react-inline') {
           return <LiveEdit code={literal} />
+        } else if (language === 'sh') {
```

Comment:
So... this nail is basically because prism don't know what sh is and in order for it to work if it sh i need to set the language to bash 😢